src/filters/dmx_dash.c

```diff
@@ -62,6 +62,14 @@ enum {
 	FLAG_FIRST_IN_SEG = 1<<2,
 };
 
+
+
+enum {
+    strip,
+    keep,
+    inject,
+};
+
```

Comment:
Could you please use capital, prefixed names for the values as we do in the rest of the code, e.g. BURL_KEEP,BURL_STRIP,BURL_INJECT